src/org/usfirst/frc/team4915/stronghold/commands/AutoCommand1.java

```diff
@@ -296,9 +215,9 @@ public static int getSpeed(Autonomous.Type type) {
         return speed;
     }
 
-    // Make this return true when this Command no longer needs to run execute()
     protected boolean isFinished() {
-        return false;
+        return super.isFinished(); // returns true if all the commands in this
```

Comment:
Would this fail? The super command group wouldn't ever return `true` because it waits for this command to return `true` which waits for...